src/stores/boardsNew.ts

```diff
@@ -1,22 +1,45 @@
 import type { Board } from '../api/boardsTypes'
-import { nanoid } from 'nanoid'
 import { defineStore } from 'pinia'
-import { useUserStore } from './user'
 import { ref, computed } from 'vue'
-import { db, colRef } from '../firebase'
-import { getDocs, doc, updateDoc, query, where } from 'firebase/firestore'
+import { boardsColRef } from '../firebase'
+import {
+  query,
+  where,
+  orderBy,
+  onSnapshot,
+  doc,
+  addDoc,
+  updateDoc,
+  deleteDoc,
+  serverTimestamp,
+  Query,
+  DocumentData
+} from 'firebase/firestore'
 
 export const useBoardsNewStore = defineStore('boardsNew', () => {
-  const userStore = useUserStore()
```

Comment:
Why did you remove userStore instance? 🤔

Reply:
That's where the userId should be kept and it's needed here, right?

Reply:
I don't need it in this component, because it's used only for [this function](https://github.com/Kepmon/task-manager/pull/10#discussion_r1266018063) and this function is actually invoked in the `userStore`, where I declare this property. So, I decided, it would be easier to just pass the id as a parameter to the function.